### PR TITLE
Hotfixes

### PR DESCRIPTION
**Pull request by mp-17**
Four hotfixes for bugs triggered by the dropout benchmark.

### Fixed

- `vlen_t` type bit width should allow for counting up to `MAXVL`. So we need one bit more when `MAXVL` is a power of 2.
- `MaskB queue` gives elements to the mask unit from the destination register, and it needs to take into account the `vsew` of the current operation.
- `vrf_pnt` update in the mask unit was wrong.
- Fix addressing bug in the Mask Unit, as the addressing does not depend on `vsew`.
- When `vl == 0`, valid vector instructions that do not imply CSR operations are considered as NOP.

## Checklist

- [x] Automated tests pass
- [x] Changelog updated
- [x] Code style guideline is observed

Please check our [contributing guidelines](CONTRIBUTING.md) before opening a Pull Request.

### CODE REVIEW

**Comment by suehtamacv** [comment]
Seems good! Let us wait for the CI, then update the CHANGELOG and merge it.

**Comment by mp-17** [comment]
Changelog updated, waiting for last CI run.

**Review by suehtamacv — APPROVED** [review]
LGTM, thanks for the checks, Perotti! I will merge this once the CI succeeds.

**Pull request merged**
